src/main/java/org/broadinstitute/hellbender/tools/copynumber/CreateBinningIntervals.java

```diff
@@ -0,0 +1,122 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.util.Interval;
+import htsjdk.samtools.util.IntervalList;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.VariantProgramGroup;
+import org.broadinstitute.hellbender.engine.GATKTool;
+import org.broadinstitute.hellbender.utils.IntervalUtils;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.io.File;
+import java.util.List;
+
+
+
+@CommandLineProgramProperties(
+        summary = "Split intervals into sub-interval files.",
+        oneLineSummary = "Split intervals into sub-interval files.",
+        programGroup = VariantProgramGroup.class
+)
+@DocumentedFeature
+public class CreateBinningIntervals extends GATKTool {
+    public static final String WIDTH_OF_BINS_SHORT_NAME = "bw";
+    public static final String WIDTH_OF_BINS_LONG_NAME = "binwidths";
+
+    public static final String PADDING_SHORT_NAME = "pad";
+    public static final String PADDING_LONG_NAME = "padding";
+
+    @Argument(
+            doc = "width of the bins",
+            fullName = WIDTH_OF_BINS_LONG_NAME,
+            shortName = WIDTH_OF_BINS_SHORT_NAME,
+            optional = true,
+            minValue = 1
+    )
+    private int widthOfBins = 1;
+
+    @Argument(
+            doc = "width of the padding regions",
+            fullName = PADDING_LONG_NAME,
+            shortName = PADDING_SHORT_NAME,
+            optional = true,
+            minValue = 0
+    )
+    private int padding = 0;
```

Comment:
This tool extends `GATKTool`, which means that it inherits an `IntervalArgumentCollection` that already includes a padding argument.  A new one is not needed.  BTW @samuelklee does this come up elsewhere in the CNV code?  It could be a holdover from the days of porting ReCapSeg when I feel we used to write more `CommandLineProgram`s.

Reply:
. . . and if this padding is different from the inherited padding then this demands a comment to avoid confusion.

Reply:
I don't think any of the other CNV tools need to be updated with respect to use of an `IntervalArgumentCollection`.

For this tool, see discussion elsewhere---it's possible that we may want to perform transformations before/after padding that are different than those allowed by `IntervalArgumentCollection`.  If so, we need to clearly document the order of these transformations and indicate how this padding argument is different from the `-ip` argument.

Reply:
Please check my answer to the previous comment.